cli-init.sh

```diff
@@ -2,4 +2,4 @@
 cd packages
```

Comment:
Do you need to change directory inside the script?
Also, if you _have_ to change the directory, make sure you change it back to the present working directory after the script is executed.

Reply:
thats already taken care of